$NetBSD: patch-src_Makefile.am,v 1.2 2019/01/30 01:03:47 fox Exp $

Removed tcl8.6 flags, this is now handled in the pkgsrc script.

--- src/Makefile.am.orig	2019-01-29 00:14:29.000000000 +0000
+++ src/Makefile.am
@@ -1,5 +1,5 @@
-AM_CXXFLAGS = -I ../include -I /usr/include/tcl8.6 -Wall -D_THREAD_SAFE -DHAVE_OPENGL -D__LICENSED__ -D __SVG__ -D __GPL20__ -DPREFIX=\"@prefix@\"
-LIBS +=  -ltcl8.6 -lSDL -lGL -lpthread -lasound -ljack
+AM_CXXFLAGS = -I ../include -Wall -D_THREAD_SAFE -DHAVE_OPENGL -D__LICENSED__ -D __SVG__ -D __GPL20__ -DPREFIX=\"@prefix@\"
+LIBS +=  -lSDL -lGL -lpthread -lasound -ljack
 
 AM_CFLAGS = -I ../include
 bin_PROGRAMS = din
